controllers/userController.js

```diff
@@ -2,33 +2,55 @@ const db = require("../models");
 
 // Defining methods for the booksController
 module.exports = {
-  findAll: function(req, res) {
-        db.User.find({}).then(function(dbUser) {
-            res.json(dbUser);
-        });
-    },
-  findById: function(req, res) {
-        db.User
-        .findById(req.params.id)
-        .then(dbModel => res.json(dbModel))
-        .catch(err => res.status(422).json(err));
-    },
-  create: function(req, res) {
+  findAll: function (req, res) {
+    db.User.find({}).then(function (dbUser) {
+      res.json(dbUser);
+    });
+  },
+  findById: function (req, res) {
     db.User
-    .create(req.body)
-    .then(dbModel => res.json(dbModel))
-    .catch(err => res.status(422).json(err));
-    },
-  update: function(req, res) {
-    db.User.findOneAndUpdate({ _id: req.params.id }, req.body)
-    .then(dbModel => res.json(dbModel))
-    .catch(err => res.status(422).json(err));
-},
-  remove: function(req, res) {
+      .findById(req.params.id)
+      .then(dbModel => res.json(dbModel))
+      .catch(err => res.status(422).json(err));
+  },
+  create: function (req, res) {
     db.User
-    .findById({ _id: req.params.id })
-    .then(dbModel => dbModel.remove())
-    .then(dbModel => res.json(dbModel))
-    .catch(err => res.status(422).json(err));
-    }
-};
\ No newline at end of file
+      .create(req.body)
+      .then(dbModel => res.json(dbModel))
+      .catch(err => res.status(422).json(err));
+  },
+  //   update: function(req, res) {
+  //     db.User.findOneAndUpdate({ _id: req.params.id }, req.body)
+  //     .then(dbModel => res.json(dbModel))
+  //     .catch(err => res.status(422).json(err));
+  // },
+  update: function ({params, body}) {
+    db.User.findByIdAndUpdate (
```

Comment:
UUID is used at the card level, so we will want to find the user by _id to add a card. Should also name the function to addCard or something so we know its updating that as opposed to the user info.

Reply:
mongojs.ObjectId(req.params.id)